crates/types/src/config/common.rs

```diff
@@ -422,7 +420,12 @@ impl Default for CommonOptions {
                 Some(15),
                 Some(Duration::from_secs(5)),
             ),
-            initialization_timeout: Duration::from_secs(5 * 60).into(),
+            join_cluster_retry_policy: RetryPolicy::exponential(
+                Duration::from_millis(100),
+                2.0,
+                None,
+                Some(Duration::from_secs(5)),
+            ),
```

Comment:
I am not sure about this default value. With this default value, the server will never give up. So if users misconfigure the cluster, they might never learn about it by the process giving up.